test/handlebars/standalone.test.js

```diff
@@ -0,0 +1,28 @@
+/* eslint no-console: 0 */
```

Comment:
I feel it's always safer to apply `// eslint-disable-line no-console` on the line you want to disable.

The intention of the disabling rule is to prevent any unwanted values being output where they would be exposed, so if the whole file is disabled then it increases the chances of a `console.log()` unintentionally being left in.

I appreciate this is a test file so unlikely to encounter this problem but it feels like a good habit to apply the rule only to the lines that we specifically want to disable.